src/apps/marketplace/src/app/view/CustomApp.less

```diff
@@ -3,11 +3,16 @@
 .CustomApp {
   height: calc(100vh - 54px);
   width: 100%;
+  overflow: scroll;
 
   iframe {
     height: calc(100vh - 54px);
-    width: 100%;
+    width: 130%;
```

Comment:
If you mean the parent container of this iframe, it has a 100% width.

Reply:
The reason I didn't applied 100% is because on smaller screens, the sub-app elements look compressed. 
That's why I think I should make it 130% on screens below 1600px and just have a horizontal scrollbar.